autoendpoint/src/routers/webpush.rs

```diff
@@ -177,11 +179,11 @@ impl WebPushRouter {
         &self,
         notification: &Notification,
         node_id: &str,
-    ) -> Result<Response, reqwest::Error> {
+    ) -> ApiResult<Response> {
         let url = format!("{}/push/{}", node_id, notification.subscription.user.uaid);
-        let notification = notification.serialize_for_delivery();
+        let notification = notification.serialize_for_delivery().unwrap();
```

Comment:
Forgot the last bit:
```suggestion
        let notification = notification.serialize_for_delivery()?;
```